python/tests/test_nn.py

```diff
@@ -19,6 +19,28 @@ def test_linear(self):
         outputs = layer(inputs)
         self.assertEqual(tuple(outputs.shape), (10, 8))
 
+    def test_int8_quantize(self):
```

Comment:
The test_int8_quantize test only has a basic check for correctness but not other important cases. Adding tests for different input scales/shapes could make this more robust.